pkg/runners/builtin_runner/bitwise.go

```diff
@@ -0,0 +1,99 @@
+package builtinrunner
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+const CELLS_PER_INSTANCE = 5
+const TOTAL_N_BITS = 251
+const INPUT_CELLS_PER_INSTANCE = 2
+
+type BitwiseBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func NewBitwiseBuiltinRunner(included bool) BitwiseBuiltinRunner {
+	return BitwiseBuiltinRunner{
+		base:     memory.NewRelocatable(0, 0),
```

Comment:
nit: We can omit this field when creating the struct and it will take the zero-value (which is 0,0)